pkg/Microsoft.Private.CoreFx.NETCoreApp/Microsoft.Private.CoreFx.NETCoreApp.pkgproj

```diff
@@ -88,6 +88,8 @@
 
   <ItemGroup>
     <IgnoredReference Include="System.Private.CoreLib" />
+    <IgnoredReference Include="System.Threading.Overlapped" />
```

Comment:
Those projects don't build yet so I asked Chris to just ignore them for now until we get them building and then we can remove these.